ACLPManager/manager
# upcoming: [DI-21119] - Handle jwe token limit in ACLP UI

## Description 📝
Handling jwe token limit in ACLP UI

## Changes  🔄
List any change relevant to the reviewer.
- JWE token will only be created for selected resources/database clusters.
- When JWE token is getting fetched, we will show graphs loading.

## Target release date 🗓️
--

## Preview 📷
**Include a screenshot or screen recording of the change.**

:lock: Use the [Mask Sensitive Data](https://cloud.linode.com/profile/settings) setting for security.

:bulb: Use `<video src="" />` tag when including recordings in table.

| Before  | After   |
| ------- | ------- |
| 📷 | 📷 |

## How to test 🧪

### Verification steps
(How to verify changes)
- Navigate to monitor tab.
- Select all the filters - dashboard, region, and resources.
- Observe the graphs loading when the jwe token is getting fetched.
- Open networks tab, go to the token api call, observe the payload - only selected resource ids have been passed.

## As an Author I have considered 🤔

*Check all that apply*
